languages/swift/exercises/concept/structs-and-classes/.docs/instructions.md

```diff
@@ -0,0 +1,57 @@
+- `move(to:)` : `(Position) -> ()` - This is similar to `resize(to:)`, however, this method adjusts the _position_ of the window to the requested value, rather than the size. As with `resize` the new position cannot exceed certain limits. - The smallest position is 0 for both `x` and `y`. - The maximum position in either direction depends on the current size of the window; the edges cannot move past the edges of the screen. Values larger than these bounds will be clipped to the largest size they can take. E.g. if the window's size is at `x` = 250, `y` = 100 and a move to `x` = 600, `y` = 200 is requested, then the window would be moved to `x` = 550, `y` = 200 as the screen is not large enough in the `x` direction to fully accommodate the request.
+- `move(deltaX:deltaY)` : `(Int, Int) -> ()` - This is similar to `resize(deltaW:deltaH)`, however, this method adjusts the _position_ of the window to the requested value, rather than the size. As with `resize` the new position cannot exceed certain limits. - The smallest position is 0 for both `x` and `y`. - The maximum position in either direction depends on the current size and position of the window; the edges cannot move past the edges of the screen. Values larger than these bounds will be clipped to the largest size they can take. E.g. if the window's current size is `width` = 150, height = 150 and its current position is at `x` = 500, `y` = 450, and a move with `deltaX` = -600, `y` = 200 is requested, then the window would be moved to `x` = 0, `y` = 450.
+- `update(title:)` : `(String) -> ()` - This method sets the `title` property to the value of the string that was passed in.
+- `update(text:)` : `(String?) -> ()` - This method sets the `contents` property to the value of the optional string that was passed in.
+- `display()` : `() -> String` - This method returns a string describing the current state of the window. For example, if the window has the `title` "My First Window" with position: x = 10, y = 100; size: width = 200, height = 150; and contents: "I 😍 my window", it should return the string: `"My First Window\nPosition (10, 100), Size: (200 x 150)\nI 😍 my window\n"` - If `contents` is nil, the last line should read "[This window intentionally left blank]"
```

Comment:
This is quite a bit of information. As a student, I think I would find this a bit confusing. Could you perhaps split this up a bit into separate tasks? It might also be worth to check if there are perhaps things that can be omitted, taking into account the fact that proficient Swift students should be able to solve the exercise in 5-10 minutes (that includes reading the introduction and instructions).

Reply:
Broken into multiple tasks. Removed double move and resize methods.